.github/workflows/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 name: ci
 
+# yamllint disable-line rule:truthy
```

Comment:
What does this line do?

Reply:
Nevermind, I see it now. yamllint, whether rightly or not, is seeing the `on` field name and warning that a value of `on` would result in a boolean value. In my opinion yamllint should't complain about field names but it is so this convinces it to stop.